Request for support for Prisma 5.x

Using the latest version of nx-prisma (5.0.2) and the latest prisma (5.1.1.) causes the error below during `npm i`.  

```
npm ERR! While resolving: @nx-tools/nx-prisma@5.0.2
npm ERR! Found: prisma@5.1.1
npm ERR! node_modules/prisma
npm ERR!   dev prisma@"^5.1.1" from the root project
npm ERR!   peerOptional prisma@"*" from @prisma/client@5.1.1
npm ERR!   node_modules/@prisma/client
npm ERR!     @prisma/client@"^5.1.1" from the root project
npm ERR!     peer @prisma/client@"^2.0.0 || ^3.0.0 || ^4.0.0 || ^5.0.0" from @devoxa/prisma-relay-cursor-connection@3.1.0
npm ERR!     node_modules/@devoxa/prisma-relay-cursor-connection
npm ERR!       @devoxa/prisma-relay-cursor-connection@"^3.1.0" from the root project
npm ERR!     1 more (nestjs-prisma)
npm ERR!   1 more (nestjs-prisma)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer prisma@"^4.5.0" from @nx-tools/nx-prisma@5.0.2
npm ERR! node_modules/@nx-tools/nx-prisma
npm ERR!   dev @nx-tools/nx-prisma@"^5.0.2" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: prisma@4.16.2
npm ERR! node_modules/prisma
npm ERR!   peer prisma@"^4.5.0" from @nx-tools/nx-prisma@5.0.2
npm ERR!   node_modules/@nx-tools/nx-prisma
npm ERR!     dev @nx-tools/nx-prisma@"^5.0.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /home/kenono/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
```

--- a/packages/nx-prisma/package.json
+++ b/packages/nx-prisma/package.json
@@ -18,7 +18,7 @@
   "peerDependencies": {
     "@nx/devkit": "^16.0.0",
     "@nx/workspace": "^16.0.0",
-    "prisma": "^4.5.0",
+    "prisma": "^4.5.0 || ^5.0.0",
     "ts-node": "*",
     "tslib": "^2.5.3"
   }

--- a/packages/nx-prisma/src/generators/init/generator.ts
+++ b/packages/nx-prisma/src/generators/init/generator.ts
@@ -18,7 +18,7 @@ export default async function (tree: Tree, options: InitGeneratorSchema): Promis
     }
   }
 
-  addPrismaConfig(tree, options, project, prismaInstalledVersion ?? '4');
+  addPrismaConfig(tree, options, project);
 
   updateProjectConfiguration(tree, options.project, {
     ...project,

--- a/packages/nx-prisma/src/generators/init/lib/add-prisma-config.ts
+++ b/packages/nx-prisma/src/generators/init/lib/add-prisma-config.ts
@@ -1,12 +1,7 @@
 import { generateFiles, joinPathFragments, ProjectConfiguration, stripIndents, Tree } from '@nx/devkit';
 import { InitGeneratorSchema } from '../schema';
 
-export function addPrismaConfig(
-  tree: Tree,
-  options: InitGeneratorSchema,
-  project: ProjectConfiguration,
-  prismaVersion: '3' | '4'
-): void {
+export function addPrismaConfig(tree: Tree, options: InitGeneratorSchema, project: ProjectConfiguration): void {
   if (tree.exists(joinPathFragments(project.root, 'prisma', 'schema.prisma'))) {
     throw new Error(
       stripIndents`The "schema.prisma" file already exists in the project "${options.project}". Are you sure this is the right project to set up Prisma?
@@ -14,7 +9,6 @@ export function addPrismaConfig(
     );
   }
 
-  // const filesDir = prismaVersion === '4' ? 'files/v4' : 'files/v3';
   const filesDir = 'files/v4';
 
   generateFiles(tree, joinPathFragments(__dirname, '..', filesDir), joinPathFragments(project.root, 'prisma'), {});

--- a/packages/nx-prisma/src/generators/init/lib/detect-prisma-installed-version.ts
+++ b/packages/nx-prisma/src/generators/init/lib/detect-prisma-installed-version.ts
@@ -2,7 +2,7 @@ import { readJson, Tree } from '@nx/devkit';
 import { checkAndCleanWithSemver } from '@nx/workspace/src/utilities/version-utils';
 import { lt } from 'semver';
 
-export function detectPrismaInstalledVersion(tree: Tree): '3' | '4' | undefined {
+export function detectPrismaInstalledVersion(tree: Tree): '4' | '5' | undefined {
   const { dependencies, devDependencies } = readJson(tree, 'package.json');
   const prismaVersion = dependencies?.['@prisma/client'] ?? devDependencies?.['@prisma/client'];
 
@@ -15,5 +15,5 @@ export function detectPrismaInstalledVersion(tree: Tree): '3' | '4' | undefined 
     throw new Error(`The prisma version "${prismaVersion}" is not supported. Please upgrade to v4.0.0 or higher.`);
   }
 
-  return lt(version, '4.0.0') ? '3' : '4';
+  return lt(version, '5.0.0') ? '4' : '5';
 }

--- a/packages/nx-prisma/src/generators/init/utils/versions.ts
+++ b/packages/nx-prisma/src/generators/init/utils/versions.ts
@@ -1,1 +1,1 @@
-export const prismaVersion = '^4.5.0';
+export const prismaVersion = '^5.1.1';

